recipe/Makefile.conda.PAR

```diff
@@ -29,12 +29,13 @@ LIBEXT  = .a
 OUTC    = -o
 OUTF    = -o
 RM = /bin/rm -f
-CC ?= gcc -fPIC
+CC = gcc
```

Comment:
This could be replaced by the line below. Then the `sed` is unneeded as well. `toolchain` already sets `CC=clang` or gcc as appropriate, and that's exactly what we don't want to override.